mweb.html

```diff
@@ -52,6 +52,23 @@
 	</script>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 	<link rel="stylesheet" href="style.css">
+	<!-- Snap Pixel Code -->
+<script type='text/javascript'>
+	(function(e,t,n){if(e.snaptr)return;var a=e.snaptr=function()
+	{a.handleRequest?a.handleRequest.apply(a,arguments):a.queue.push(arguments)};
+	a.queue=[];var s='script';r=t.createElement(s);r.async=!0;
+	r.src=n;var u=t.getElementsByTagName(s)[0];
+	u.parentNode.insertBefore(r,u);})(window,document,
+	'https://sc-static.net/scevent.min.js');
+	
+	snaptr('init', '87216ec4-f9a2-42ae-8147-0e4df333c206', {
+	'user_email': '__INSERT_USER_EMAIL__'
```

Comment:
@zeeshanMhussain is it mandatory to pass an email ID to Snap here?

Reply:
It is part of pixel snippet they provide.